algorithms/unit_tests/TestStdAlgorithmsTeamInclusiveScan.cpp

```diff
@@ -281,8 +281,8 @@ void run_all_scenarios() {
 
 TEST(std_algorithms_inclusive_scan_team_test, test) {
 // FIXME_OPENMPTARGET
-#if defined(KOKKOS_ENABLE_OPENMPTARGET) && defined(KOKKOS_ARCH_INTEL_GPU)
-  GTEST_SKIP() << "the test is known to fail with OpenMPTarget on Intel GPUs";
+#if defined(KOKKOS_ENABLE_OPENMPTARGET)
+  GTEST_SKIP() << "the test is known to fail with OpenMPTarget";
```

Comment:
Please resolve the conflict from merging https://github.com/kokkos/kokkos/pull/8199